tests/gold_tests/headers/cache_and_req_body_curlheader.test.py

```diff
@@ -0,0 +1,118 @@
+'''
```

Comment:
Can you modify the original test as we don't want to test the same thing twice... we just want the original test to be more reliable.